### DISCUSSION TITLE
CPI設定についてお問い合わせします。

### DISCUSSION THREAD

**Discussion by Forseti4** [Yowkees/keyball#450] [Q&A] [answered]
こんにちは。Keyball61を完成させて、非常にうまく使っています。良い製品をありがとうございます。私はファームウェアv1.2をインストールしました。CPIを変更して再度Keyballを接続すると、CPIが2800に戻ってしまいます。私が設定したCPIを保存する方法があるのでしょうか？

**Comment by Forseti4** [comment]
遅くに保存キーを見つけました。解決された状況です。

**Comment by Yowkees** [comment, marked as answer]
ご自身で見つけていただきありがとうございます。
お話しの通りCPI値の保存には、KC_SAVEをご使用いただければと思います。

以上、よろしくお願いいたします。
